polaris/polaris/management/commands/poll_pending_deposits.py

```diff
@@ -32,6 +32,7 @@ def execute_deposit(transaction: Transaction) -> bool:
     valid_statuses = [
         Transaction.STATUS.pending_user_transfer_start,
         Transaction.STATUS.pending_anchor,
+        Transaction.STATUS.pending_trust,
```

Comment:
Transaction shouldn't be `pending_trust` when this is called. If CBs are supported by the wallet, then its the transaction should be `pending_anchor` because we don't need to wait for the trustline.